docs/installation/kubernetes.md

```diff
@@ -0,0 +1,138 @@
+---
+id: kubernetes
+title: Kubernetes
+---
+
+## Kubernetes
+
+If you are using Kubernetes you may find the following scripts useful for a new installation of Pitchfork.
+
+They consist of a deployment, a service and a horizontal pod autoscaler that you can modify according to your needs.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pitchfork
+  labels:
+    app: pitchfork
+spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pitchfork
+  template:
+    metadata:
+      labels:
+        app: pitchfork
+        release: pitchfork
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8081"
+        prometheus.io/path: "/actuator/prometheus"
+    spec:
+      terminationGracePeriodSeconds: 0
```

Comment:
I think this needs to be > preStop hook sleep duration. By default it's 30s.
Kubernetes waits for time=terminationGracePeriodSeconds and this happens in parallel to whatever we set in the preStop hook.

Reply:
yes, you're right. i'll remove this line.